### PR TITLE
feat: add ioxQuery limits

### PR DESCRIPTION
**Pull request by mhilton**
Add support for viewing and modifying the new IOx Query limits.

Closes #

### Checklist

Authors and Reviewer(s), please verify the following:

- [x] A PR description, regardless of the triviality of this change, that communicates the value of this PR
- [x] [Well-formatted conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) that provide context into the change
- [ ] Documentation updated or issue created (provide link to issue/PR)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [ ] Feature flagged, if applicable

### CODE REVIEW

**Comment by appletreeisyellow** [comment]
Is it possible to add a before and after screenshot in the description? Something like this https://github.com/influxdata/ui/pull/6657

File: src/operator/OrgOverlay.tsx [hunk 1]
```diff
@@ -339,6 +339,28 @@ export const OrgOverlay: FC = () => {
                       </Grid.Column>
                     </Grid.Row>
                   )}
+                  {limits?.ioxQuery && (
```
Comment (appletreeisyellow) [inline src/operator/OrgOverlay.tsx:342]: Seems like [ioxQuery](https://github.com/influxdata/openapi/blob/3380e8851d26ca33131a3e183d7147b42fdf2a09/contracts/priv/unity.yml#L2738) is defined under the type [OrgLimits](https://github.com/influxdata/openapi/blob/35ecbc2e2890e77756d0be0f801b9e503a8ffd44/contracts/priv/quartz-oem.yml#L471), but `limits` is a type of [OperatorOrgLimits](https://github.com/influxdata/openapi/blob/3380e8851d26ca33131a3e183d7147b42fdf2a09/contracts/priv/unity.yml#L1890)

Comment (wdoconnell) [reply src/operator/OrgOverlay.tsx:342]: I think this is now resolved after https://github.com/influxdata/openapi/pull/649

**Comment by wdoconnell** [comment]
> Is it possible to add a before and after screenshot in the description? Something like this #6657

@mhilton and I discussed this: Given the current implementation for mocking these limits, the addition of this limit on the operator page will not reflect meaningful data in local dev. I will add more details in the 🧵 , but staging is the better place to revalidate this.

**Review by wdoconnell — APPROVED** [review]

**Pull request merged**